google/api_core/retry_async.py

```diff
@@ -68,9 +68,12 @@ async def check_if_exists():
 _DEFAULT_MAXIMUM_DELAY = 60.0  # seconds
 _DEFAULT_DELAY_MULTIPLIER = 2.0
 _DEFAULT_DEADLINE = 60.0 * 2.0  # seconds
+_DEFAULT_TIMEOUT = 60.0 * 2.0  # seconds
 
 
-async def retry_target(target, predicate, sleep_generator, deadline, on_error=None):
+async def retry_target(
```

Comment:
nit: any way to make the formatting nicer?

Reply:
Please do not suggest formatting, it is autoformatted